roles/openshift_facts/library/openshift_facts.py

```diff
@@ -114,6 +114,12 @@ def migrate_hosted_facts(facts):
             if 'router' not in facts['hosted']:
                 facts['hosted']['router'] = {}
             facts['hosted']['router']['selector'] = facts['master'].pop('router_selector')
+        if 'registry_selector' in facts['master']:
+            if 'hosted' not in facts:
+                facts['hosted'] = {}
+            if 'registry' not in facts['hosted']:
+                facts['hosted']['registry'] = {}
+            facts['hosted']['registry']['selector'] = facts['master'].pop('registry_selector')
```

Comment:
Is this a situation where we can just skip openshift_facts entirely? From what I can see this isn't really a fact *about* the system, we're configuring a desired state based either on what's in the inventory, or a default. Would it be possible to just ditch this entirely in openshift_facts, avoid transfering the variable to fact, upgrading old facts, and instead just use openshift_hosted_registry_selector, or a default.